src/locales/en/translation.json

```diff
@@ -1,13 +1,26 @@
 {
   "paymentProcessing": {
     "amount": {
-      "header": "100% of your support will go to the merchant toward their operating costs. Your transaction is secure, encrypted and directly goes to the owner for sustaining their business.",
+      "donation": {
+        "header": "Donation for",
+        "body": "Your donation is secure, encrypted, and directly goes to the restaurant owner for sustaining their business."
+      },
+      "voucher": {
+        "header": "Voucher for",
+        "body": "Your payment is secure, encrypted, and directly goes to the restaurant owner for sustaining their business."
+      },
+      "cover": {
+        "body1": "When you donate via credit card, there are usually payment processing fees of 2-3% on the transaction. If you make a $100 donation, the merchant only gets $97 of it.",
```

Comment:
We should make this copy more succinct. I think we can remove the 2-3% and just say 3%. I think 3% is easy to wrap your mind around, even though it's actually 2.9% + $.30. cc @nanxiy

Reply:
I'd also remove "usually", since there is always a processing fee

Reply:
FYI the merchant donate page currently has "100% of all proceeds go to the businesses, and we cover the credit card processing fees." - which contradicts what we are proposing

Reply:
@nanxiy can we remove the "and we cover the credit card processing fees" part? 100% of proceeds still do go to the merchant

Reply:
yeah let's remove it - Chinese version will need to be updated too, will let Chel know

Reply:
@nanxiy Hmm. So what should the final copy be? 

`cover.body1`
- remove "usually"
- just write "3%" instead of "2-3%"
- remove "via credit card"

`contributionBar.footer`
- Remove "and we cover the credit card processing fees"

Hmm. Also, would this copy appear a bit contradictory if we state "3%", but when the $100 option is selected, the customer sees transaction fees are "$3.20"?  Likewise with the lower amounts.

Reply:
"When you donate, there are payment processing fees of about 3% on the transaction. If you make a $100 donation, the merchant only gets $97 of it."

^ Added "about" to the copy. We can either say 'about' and do cents, or just do 3% and then round to the dollar. I think I prefer the former

Reply:
Agreed, I like the former better too. I'll make the changes, thanks! 👍

Reply:
@nanxiy @Kenworthe After looking at this PR and seeing the confusion in copy, I think we should do one of two things:
1. Always apply the 3% fee on top of the donation amount. The average donation is $5-$10 right now, which is at most a 60 cent fee, which to the donor is negligible. So if they select $10, they'll be charged $10.60.
2. Always apply the 3% fee and take it away from the donation amount. In this case, they'll be still charged $10, but in a breakdown somewhere we'll show them that the X will go to the merchant, and Y will go towards credit card processing fees.

Ultimately we want donations to be as fast and easy as possible, and reducing the amount of mental effort that needs to be taken when making a donation is important — while still keeping things transparent. At the end of the day, option 1 and 2 are the same thing, it's just a matter of perspective. 

@nanxiy would you be alright with revisiting the design of this a tad and @Kenworthe sorry for the thrash on the PR. We should have caught this in Figma before it got to this stage

Reply:
@justintmckibben okay as discussed earlier, I just made the changes to the precise transaction fee formula, rounded up to the nearest cent. left copy alone for now - we can do that as a fast follow.

Reply:
@justintmckibben Can you clarity revisiting which part of the design ? did we decided if we were going with 1 or 2? I think if we go with 1 then the design stands, which is what Ken is working on right?

Reply:
I think that we need to revisit design here. The mocks are confusing, and are basically making the customer choose between two options that essentially do the same thing. I think we should remove the copy and the ability to choose, and instead just give the customer a breakdown of where the money is going upon checkout. Fundamentally, giving the customer this option doesn't really make any difference. 

We should either: 
1. Always charge the customer the fee, and tell them that they're being charged (like we do taxes upon checkout)
2. Always take the fee out of the amount the customer has selected to donate (which will be surfaced in a breakdown similar to taxes)

Reply:
https://www.figma.com/file/I0XPeX6re8t14J4nZOvVNp/Send-Chinatown-Love?node-id=104%3A844